configs/wider_face/ssd300_wider_face.py

```diff
@@ -107,7 +107,7 @@
         img_prefix=data_root + 'WIDER_val/',
         pipeline=test_pipeline))
 # optimizer
-optimizer = dict(type='SGD', lr=1e-3, momentum=0.9, weight_decay=5e-4)
+optimizer = dict(type='SGD', lr=8*1.5e-3, momentum=0.9, weight_decay=5e-4)
```

Comment:
To be consistent with other configs, maybe just `0.02` instead of `8*1.5e-3`?

Reply:
Ok, but 8*1.5e-3 = 0.012